CHANGES.txt

```diff
@@ -2,6 +2,86 @@
 Pyface Changelog
 ================
 
+Release 7.1.0
+=============
+
+Released: 2020-10-XX
```

Comment:
We have not been including release date. This is probably not necessary given that both PyPI and GitHub present the date attached with the release and those should be more accurate.